osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySliderBody.cs

```diff
@@ -23,29 +23,32 @@ protected override Color4 GetBodyAccentColour(ISkinSource skin, Color4 hitObject
 
         private partial class LegacyDrawableSliderPath : DrawableSliderPath
         {
-            private const float shadow_portion = 1 - (OsuLegacySkinTransformer.LEGACY_CIRCLE_RADIUS / OsuHitObject.OBJECT_RADIUS);
-
-            protected new float CalculatedBorderPortion
-                // Roughly matches osu!stable's slider border portions.
-                => base.CalculatedBorderPortion * 0.77f;
-
             protected override Color4 ColourAt(float position)
             {
-                float realBorderPortion = shadow_portion + CalculatedBorderPortion;
-                float realGradientPortion = 1 - realBorderPortion;
+                // https://github.com/peppy/osu-stable-reference/blob/3ea48705eb67172c430371dcfc8a16a002ed0d3d/osu!/Graphics/Renderers/MmSliderRendererGL.cs#L99
+                const float aa_width = 0.5f / 64f;
```

Comment:
should be 3/256, not 0.5/64

Reply:
Does it? ppy posted the `aa_width` definition in this thread, and that one says it's 3/256.